Casks/xbar.rb

```diff
@@ -0,0 +1,24 @@
+cask "xbar" do
+  version "2.0.32-beta"
+  sha256 "3fbce5a0acd46ad227f269245a5e4d8acf44c84a322848da1e4cfe252f3decf3"
+
+  url "https://github.com/matryer/xbar/releases/download/v#{version}/xbar.v#{version}.dmg",
+      verified: "github.com/matryer/xbar/"
+  name "xbar"
+  desc "Put anything in your menu bar"
```

Comment:
From the [`desc` documentation](https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/desc.md):

> `desc` is not for app slogans! Vendors’ descriptions tend to be filled with generic adjectives such as “modern” and “lightweight”. Those are meaningless marketing fluff (do you ever see apps proudly describing themselves as outdated and bulky?) which must the deleted. It’s fine to use the information on the software’s website as a starting point, but it will require editing in almost all cases.

```suggestion
  desc "View output from scripts in the menu bar"
```